mkdocs.yml

```
@@ -5,7 +5,7 @@ nav:
   - Home: index.md
   - Installation: buildinstall.md
   - Python API: api/python/isce3/core.html
```

Not related to the changes in this PR, but I noticed that when I click on "Python API" link from the landing page, it takes me directly to the API docs for the `isce3.core` submodule, instead of the top-level `isce3` module.

I think it would make sense to change this to point users to the API docs for the top-level module. Otherwise, folks who aren't familiar with the API might be confused and think the library is a lot smaller than it is. WDYT?

```suggestion
  - Python API: api/python/isce3.html
```

---

Somewhat relatedly, we should probably find some way to document the API of the `nisar` package as well, if possible.

I suppose that would require another call to `pdoc`?

Then, I could imagine that we might want to include links to both the `isce3` API docs and the `nisar` API docs in this navigation sidebar. Maybe it would look something like this?

```suggestion
  - Python API:
      - isce3: api/python/isce3.html
      - nisar: api/python/nisar.html
```